frontend/src/themes/base/globals.js

```diff
@@ -159,7 +159,7 @@ const globals = {
   '.ag-theme-quartz .ag-header-cell-label': {
     justifyContent: 'center'
   },
-  '.ag-theme-quartz .ag-cell-inline-editing .ag-cell:focus, .ag-input-field-input:focus, .ag-picker-field-wrapper:focus':
```

Comment:
This allows disabled cells to still show a border when they are focused, as disabled is not the same as not focusable.